cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,25 @@
 /// <reference types="cypress" />
 
+beforeEach('My home page', () => {
```

Comment:
1. Better to nest beforeEach inside describe 
2. beforeEach can be without title - beforeEach(() => {})